docs/standard/translation/workflow.rst

```diff
@@ -3,10 +3,13 @@ Translation workflow
 
 .. _glossary:
 
-Glossary
---------
+Glossary and conventions
```

Comment:
This glossary section and the conventions in the Google document can be moved to quality.rst, which should probably be moved to after the Workflow in the toctree. Anyway, for this PR, we can just fix the glossary section where it is currently.

```suggestion
Glossary
```